### DISCUSSION TITLE
cannot get "twitter card" working

### DISCUSSION THREAD

**Discussion by produnis** [quarto-dev/quarto-cli#4211] [Q&A] [answered]
Hi folks,
I have problems setting up a "twitter card".

My website is https://www.produnis.de/R  and my yaml can be found here  https://pastebin.com/x8bkb26N

I don't know what I am doing wrong, but if I copy a link from my website inside any messanger, no preview-pic will be shown...

What am I missing?

thx in advance

**Comment by mcanouil** [comment]
What's wrong exactly? 🤔

Note that OpenGraph and Twitter Card require cache to be refreshed and some time to propagate.

<img width="590" alt="image" src="https://user-images.githubusercontent.com/8896044/216713027-a2233d00-2e3e-4ec8-ba81-54d847f77b2c.png">

**Comment by produnis** [comment]
thats strange... at my side, the pic is not shown, instead, I see the placeholder....

![](https://i.imgur.com/kWmt0ZW.png)

**Comment by produnis** [comment, marked as answer]
ok, on twitter it is working, on mastodon and other messangers not..... ;/

**Reply by mcanouil** [reply]
Twitter card is different from OpenGraph.

Twitter card is only for Twitter.

And as I said, it needs time to propagate and for cache to be automatically invalidated and updated.

**Reply by produnis** [reply]
Ah, ok, I didnt know what Opencard was.... I'll dive into it.....

**Reply by mcanouil** [reply]
FYI, it's all documented on Quarto website, see <https://quarto.org/docs/websites/website-tools.html#open-graph>. 😉
